parallel-consumer-core/src/main/java/io/confluent/parallelconsumer/state/RemovedPartitionState.java

```diff
@@ -42,7 +44,7 @@
     public static final int NO_EPOCH = -1;
 
     public RemovedPartitionState() {
-        super(NO_EPOCH, null, null, OffsetMapCodecManager.HighestOffsetAndIncompletes.of());
+        super(NO_EPOCH, new PCModule<>(ParallelConsumerOptions.<K, V>builder().build()), null, OffsetMapCodecManager.HighestOffsetAndIncompletes.of());
```

Comment:
shouldn't be building new modules - make constructor private? restrict somehow? have had problems with multiple modules in the past sneaking their way in